skopt/utils.py

```diff
@@ -501,3 +499,104 @@ def normalize_dimensions(dimensions):
                                    "(%s)" % type(dimension))
 
     return Space(transformed_dimensions)
+
+
+def wrap_named_args(dimensions):
+    """
+    Wrapper for an objective function that uses named arguments to make it
```

Comment:
Wrapper -> Decorator? That's the standard name for that sort of things in Python.

Reply:
I think you are deliberately nitpicking here and although I'm not a Python expert I doubt that what you're saying is strictly true. We are importing and using something called *wraps* from `functools` and they repeatedly talk about *wrapping* functions in the documentation, and they even call it `wrapper` in their code examples. Yes, these are also called function decorators in Python when used with the @ symbol. But this code may be used by people who are not Python experts so they may not know what a decorator is, but they most likely know what a function-wrapper is.

I'll tell you what. I'll call it a Wrapper / Decorator so everyone is happy :-) Besides I think `use_named_args` is a better name as well.

https://docs.python.org/2/library/functools.html